llvm/utils/lit/tests/shtest-cat.py

```diff
@@ -0,0 +1,20 @@
+## Check the cat command.
```

Comment:
Should this test check the passed/failed summary like the shtest-shell test does?

Reply:
Probably. It's likely to catch out tests that are added, but didn't update this file correctly. I'd expect it would be possible to miss them otherwise.